Makefile

```diff
@@ -90,8 +94,22 @@ kcm-dist-release: helm yq
 templates-generate:
 	@hack/templates.sh
 
+CAPO_DIR := $(PROVIDER_TEMPLATES_DIR)/cluster-api-provider-openstack
+CAPO_ORC_VERSION ?= $(shell grep 'orcVersion:' $(CAPO_DIR)/values.yaml | cut -d '"' -f 2)
+CAPO_ORC_TEMPLATE := "$(CAPO_DIR)/templates/orc-$(shell echo $(CAPO_ORC_VERSION) | sed 's/\./-/g').yaml"
+.PHONY: capo-orc-fetch
+capo-orc-fetch: yq
+	@if test -s "$(CAPO_ORC_TEMPLATE)"; then \
+		echo "ORC installation file with version $(CAPO_ORC_VERSION) already exists in $(CAPO_DIR) chart."; \
+	else \
+	  	curl -L --fail -s https://github.com/k-orc/openstack-resource-controller/releases/download/v$(CAPO_ORC_VERSION)/install.yaml -o $(CAPO_ORC_TEMPLATE); \
+	  	$(YQ) -i '(select(.kind == "Deployment") | .spec.template.spec.containers.[0].image) |= sub("$(CAPO_ORC_VERSION)"/"{{ .Values.orcVersion }}")' $(CAPO_ORC_TEMPLATE); \
+	  	sed  -i '1i {{ if (eq .Values.orcVersion "$(CAPO_ORC_VERSION)") }}' $(CAPO_ORC_TEMPLATE); \
```

Comment:
this will not work on bsd-based `sed` binary, in particular, darwin. since not all of us has `gsed` (especially substituted in PATH). i will try to find another approach and then suggest it

Reply:
that should work
```makefile
...
		printf '{{ if (eq .Values.orcVersion "$(CAPO_ORC_VERSION)") }}\n' > $(CAPO_ORC_TEMPLATE).bak && \
		cat $(CAPO_ORC_TEMPLATE) >> $(CAPO_ORC_TEMPLATE).bak && \
		mv $(CAPO_ORC_TEMPLATE).bak $(CAPO_ORC_TEMPLATE) && \
```

whether to put `&&` everywhere or a plain `; ` is up to you